paasta_tools/docker_wrapper.py

```diff
@@ -272,6 +276,44 @@ def append_cpuset_args(argv, env_args):
     return argv
 
 
+def add_firewall(argv, service, instance):
+    output = ''
+    sensu_status = pysensu_yelp.Status.OK
+
+    try:
+        mac_address, lockfile = reserve_unique_mac_address(LOCK_DIRECTORY)
+    except Exception as e:
+        output = 'Unable to add mac address: {}'.format(e)
+        sensu_status = pysensu_yelp.Status.CRITICAL
+    else:
+        argv = add_argument(argv, '--mac-address={}'.format(mac_address))
+        try:
+
+            with firewall_flock():
+                prepare_new_container(
+                    DEFAULT_SOA_DIR,
+                    DEFAULT_SYNAPSE_SERVICE_DIR,
+                    service,
+                    instance,
+                    mac_address)
+        except Exception as e:
+            output = 'Unable to add firewall rules: {}'.format(e)
+            sensu_status = pysensu_yelp.Status.CRITICAL
+
+    if output:
+        print(output, file=sys.stderr)
+
+    send_event(
```

Comment:
I don't have a strong opinion. I agree with @chriskuehl that it's probably easy to miss those messages. But I also agree that adding sensu introduces a bit more risk. The whole thing _is_ wrapped in a try/except, fwiw. But if you still think it should be removed, no quibbles from me.